### PR TITLE
feat: added loginform

### PR DESCRIPTION
**Pull request by DavidKorystov**

### CODE REVIEW

File: src/containers/LoginForm.tsx [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+  :hover {
+    background: rgb(15, 20, 25, 0.1);
+    transition-duration: 0.2s;
+  }
+`;
+
+interface MyForm {
+  userName: string;
+  email: string;
+  dateOfBirth: Date;
```
Comment (Mykyta-Pysarev) [inline src/containers/LoginForm.tsx]: Change type from date to string for dateOfBirth

**Comment by alexandrtovmach** [comment]
image

**Comment by DavidKorystov** [comment]
image

File: src/containers/LoginForm.tsx [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+          fill="rgb(29, 155, 240)"
+        >
+          <g>
+            <path d="M23.643 4.937c-.835.37-1.732.62-2.675.733.962-.576 1.7-1.49 2.048-2.578-.9.534-1.897.922-2.958 1.13-.85-.904-2.06-1.47-3.4-1.47-2.572 0-4.658 2.086-4.658 4.66 0 .364.042.718.12 1.06-3.873-.195-7.304-2.05-9.602-4.868-.4.69-.63 1.49-.63 2.342 0 1.616.823 3.043 2.072 3.878-.764-.025-1.482-.234-2.11-.583v.06c0 2.257 1.605 4.14 3.737 4.568-.392.106-.803.162-1.227.162-.3 0-.593-.028-.877-.082.593 1.85 2.313 3.198 4.352 3.234-1.595 1.25-3.604 1.995-5.786 1.995-.376 0-.747-.022-1.112-.065 2.062 1.323 4.51 2.093 7.14 2.093 8.57 0 13.255-7.098 13.255-13.254 0-.2-.005-.402-.014-.602.91-.658 1.7-1.477 2.323-2.41z" />
+          </g>
+        </svg>
+        <div style={{ height: 32, width: 20 }} />
+      </header>
+
+      <section style={{ width: 300 }} className="mt-4 m-auto  d-grid gap-4">
```
Comment (Mykyta-Pysarev) [inline src/containers/LoginForm.tsx]: Remove styles from element. Move it to styled components

File: src/containers/LoginForm.tsx [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+  Form,
+  OverlayTrigger,
+  Tooltip,
+  Button,
+} from "react-bootstrap";
+import { useForm } from "react-hook-form";
+import { Link } from "react-router-dom";
+import styled from "styled-components";
+
+// font-family: TwitterChirp, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif;
```
Comment (Mykyta-Pysarev) [inline src/containers/LoginForm.tsx]: Remove this comment

File: src/containers/LoginForm.tsx [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+          <div className="form-floating mb-3">
+            <input
+              {...register("email")}
+              type="email"
+              name="email"
+              className="form-control "
+              id="floatingInput"
+              placeholder="name@example.com"
+            />
+            <label>Phone, email, or username</label>
```
Comment (Mykyta-Pysarev) [inline src/containers/LoginForm.tsx:151]: As I recall, twitter has only phone and e-mail authorizations - no username.

File: src/containers/LoginForm.tsx [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+  const { register, handleSubmit } = useForm<MyForm>();
+
+  const submitForm = (data: MyForm) => {
+    return data;
+    // console.log(data);
+  };
+
+  return (
+    <Container
+      style={{ height: 650, width: 600 }}
```
Comment (KirillSerg) [inline src/containers/LoginForm.tsx]: remoov inline style

File: src/containers/LoginForm.tsx [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+  Tooltip,
+  Button,
+} from "react-bootstrap";
+import { useForm } from "react-hook-form";
+import { Link } from "react-router-dom";
+import styled from "styled-components";
+
+// font-family: TwitterChirp, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif;
+
+const OnHoverClose = styled.div`
```
Comment (KirillSerg) [inline src/containers/LoginForm.tsx]: maybe better use bootstrap component

**Review by KirillSerg — APPROVED** [review]
overall nice

File: src/containers/LoginForm.tsx [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+
+interface MyForm {
+  userName: string;
+  email: string;
+  dateOfBirth: Date;
+  password: string;
+  confirmPassword: string;
+}
+
+export const LoginForm = () => {
```
Comment (ArtemLigerko) [inline src/containers/LoginForm.tsx]: conponents usually exports as default

File: src/containers/LoginForm.tsx [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+            <div className="fs-5 text-secondary mx-2">or</div>
+            <div className="bg-secondary border-bottom border-secondary w-50" />
+          </div>
+
+          <div className="form-floating mb-3">
+            <input
+              {...register("email")}
+              type="email"
+              name="email"
+              className="form-control "
```
Comment (ArtemLigerko) [inline src/containers/LoginForm.tsx]: remove space

File: src/containers/LoginForm.tsx [hunk 9]
```diff
@@ -0,0 +1,179 @@
...
+            <svg
+              viewBox="0 0 24 24"
+              aria-hidden="true"
+              width="20px"
+              height="20px"
+            >
+              <g>
+                <path d="M13.414 12l5.793-5.793c.39-.39.39-1.023 0-1.414s-1.023-.39-1.414 0L12 10.586 6.207 4.793c-.39-.39-1.023-.39-1.414 0s-.39 1.023 0 1.414L10.586 12l-5.793 5.793c-.39.39-.39 1.023 0 1.414.195.195.45.293.707.293s.512-.098.707-.293L12 13.414l5.793 5.793c.195.195.45.293.707.293s.512-.098.707-.293c.39-.39.39-1.023 0-1.414L13.414 12z" />
+              </g>
+            </svg>
```
Comment (AndrewKaenko) [inline src/containers/LoginForm.tsx:63]: Instead, you can use an icons from a third-party library

Comment (DavidKorystov) [reply src/containers/LoginForm.tsx:63]: This will be realized if another member adds this library in his task

**Comment by alexandrtovmach** [comment]
merge conflicts

**Review by alexandrtovmach — APPROVED** [review]

**Pull request merged**